app/src/main/java/com/example/homework6/koin/RepositoryModule.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.homework6.koin
+
+import com.example.homework6.model.repositories.PersonRepositoryImpl
+import org.koin.dsl.module
+
+val repositoryModule = module {
+
+    single {
+        PersonRepositoryImpl(dao = get(), api = get())
+    }
```

Comment:
здесь неободимо дополнительно параметризировать, чтобы не было возможности пользоваться реализацией
```suggestion
    single<PersonRepository> {
        PersonRepositoryImpl(dao = get(), api = get())
    }
```